.github/ISSUE_TEMPLATE/bug.md

```diff
@@ -0,0 +1,21 @@
+---
+name: 🐞 Bug Report
+about: Report an issue with kpt
+title: ""
+labels:
+  - bug
+assignees: ""
+---
+
+### Expected behavior
+
+### Actual behavior
+
+### Information
+
+<!-- Please include any additional information related to your issue -->
```

Comment:
is this comment rendered and viewable by the user?

Reply:
Yes. Everything on this yaml page is rendered into the Issue's details. Here's the issue experience in kustomize which has similar templates: https://github.com/kubernetes-sigs/kustomize/issues/new/choose

If the comments are left in the issue they should not be rendered in the filed issues.